src/types/schemas/Note.ts

```diff
@@ -0,0 +1,16 @@
+export default interface Note {
+  /**
+   * creation date of the message in timestamp format
+   */
+  creationDate: Date
```

Comment:
Make this property optional as some services don't always provide it (google)

Reply:
Fixed